query/iam/iam_subscription_owner_max_3.sql

```diff
@@ -0,0 +1,32 @@
+with owner_roles as (
+  select
+    d.role_name,
+    d.role_type,
+    d.name,
+    d.title,
+    d.subscription_id,
+    a.principal_id as principal_id
+  from
+    azure_role_definition as d
+    left join azure_role_assignment as a on d.id = a.role_definition_id
+  where
+    d.role_name = 'Owner'
```

Comment:
Are we also including `Service Principals` in this check?
As per the docs, we should exclude `Service Principals`.

Reply:
we are not using the principals as part of query. it was used only in the select part, hence removed it from the select part as well.